profile/README.md

```diff
@@ -0,0 +1,21 @@
+# CROCODILE: Python interfaces for regional ocean modeling and data assimilation
+
+### CROCODILE is an NSF-funded cyberinfrastructure collaboration between the NSF National Center for Atmospheric Research ([NSF NCAR](https://ncar.ucar.edu/)) and the Woods Hole Oceanographic Institution ([WHOI](https://www.whoi.edu/)).
+
+### Our goal is to build a community of practice around regional ocean modeling for studying ocean physics, the carbon cycle, climate change and intervention, and more, by creating and curating a common set of open-source tools.
```

Comment:
Not sure if you've messed around with this allready and not sure we care, but there is a small font size change from ### to the next paragraph of straight text. (To keep them the same, could use **bold**  instead of ###, but I couldn't figure out how to make it pretty that way.)